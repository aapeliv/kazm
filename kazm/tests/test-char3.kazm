int main(){
	array char[5] my_arr = ['a', 'b', 'c', 'd', 'e'];
	//rray string[5] my_arr = ["hi", "katie", "aapeli", "zhonglin", "molly"];
	int i;
	for (i = 0; i<my_arr.length ;i = i+1){
		char_println(my_arr[i]);
	}
	
	int x = my_arr.length;  
	int_println(x);
}
