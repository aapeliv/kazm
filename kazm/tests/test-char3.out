
a
b
c
d
e
5